client/src/components/menubar/index.tsx

```diff
@@ -21,6 +21,11 @@ import { GRAPH_AS_IMAGE_TEST_ID } from "../../util/constants";
 import { AppDispatch, RootState } from "../../reducers";
 import { AnnoMatrixClipView } from "../../annoMatrix/views";
 
+const INITIAL_PERCENTILES = {
```

Comment:
this file change is to fix the bug raised here:

https://czi-sci.slack.com/archives/C0247APK621/p1721404729785679?thread_ts=1721165771.450419&cid=C0247APK621